### PR TITLE
fix: replace jsonparser with fastjson to delete exported variables

### PR DESCRIPTION
**Pull request by jensneuse**
- replace jsonparser.Delete with fastjson
- limit concurrency of parse kit to num CPUs

### CODE REVIEW

File: aws-lambda-router/go.mod [hunk 1]
```diff
@@ -89,17 +95,17 @@ require (
...
 	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v0.44.0 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.23.1 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.23.1 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.23.1 // indirect
-	go.opentelemetry.io/otel/exporters/prometheus v0.46.0 // indirect
-	go.opentelemetry.io/otel/metric v1.24.0 // indirect
-	go.opentelemetry.io/otel/sdk v1.24.0 // indirect
-	go.opentelemetry.io/otel/sdk/metric v1.24.0 // indirect
-	go.opentelemetry.io/otel/trace v1.24.0 // indirect
+	go.opentelemetry.io/otel/exporters/prometheus v0.50.0 // indirect
```
Comment (StarpTech) [inline aws-lambda-router/go.mod:104]: Why was it upgraded for no reasons?

File: router/pkg/config/config.schema.json [hunk 2]
```diff
@@ -1420,6 +1420,11 @@
           "type": "integer",
           "default": 1024,
           "description": "The size of the normalization cache."
+        },
+        "parse_kit_pool_size":{
```
Comment (StarpTech) [inline router/pkg/config/config.schema.json:1424]: parsekit_pool_size ?

File: router/go.mod [hunk 3]
```diff
@@ -44,24 +44,24 @@ require (
...
 	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v0.44.0
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.23.1
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.23.1
-	go.opentelemetry.io/otel/exporters/prometheus v0.46.0
-	go.opentelemetry.io/otel/metric v1.24.0
-	go.opentelemetry.io/otel/sdk v1.24.0
-	go.opentelemetry.io/otel/sdk/metric v1.24.0
-	go.opentelemetry.io/otel/trace v1.24.0
+	go.opentelemetry.io/otel/exporters/prometheus v0.50.0
+	go.opentelemetry.io/otel/metric v1.28.0
```
Comment (StarpTech) [inline router/go.mod:53]: This is quite dangerous. Why was it upgraded?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: router/pkg/config/config.schema.json (hunk 2 above)
Comment (jensneuse) [reply router/pkg/config/config.schema.json]: ok

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**